### PR TITLE
Ross branch

### PR DESCRIPTION
**Pull request by TomNeo**
Testing the pull request stuff

### CODE REVIEW

File: core/src/net/heinousgames/game/dungeoncrawler/TestMain.java [hunk 1]
```diff
@@ -0,0 +1,335 @@
...
+
+    @Override
+    public void create() {
+
+        // player class defined above
+        player = new Player();
+        // put him in the bottom corner
+        player.pos.set(0, 0);
+        MapLoader = new MapBuffer(new levelOne(this));
+        renderer2 = new HeinousRenderer(player,MapLoader,1/16f);
```
Comment (ministeve412) [inline core/src/net/heinousgames/game/dungeoncrawler/TestMain.java]: renderer2 = new HeinousRenderer(player, MapLoader, 1/16f);

**Comment by TomNeo** [comment]
Fuck you

**Comment by ministeve412** [comment]
👍 💯 🔥 🚀 🎸

**Comment by TomNeo** [comment]
False alarm. Pending code review

File: core/src/net/heinousgames/game/dungeoncrawler/levelOne.java [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+import com.badlogic.gdx.graphics.g2d.TextureRegion;
+import com.badlogic.gdx.maps.tiled.TiledMap;
+import com.badlogic.gdx.maps.tiled.TiledMapTileLayer;
+import com.badlogic.gdx.maps.tiled.TmxMapLoader;
+
+/**
+ * Created by User on 12/10/2016.
+ */
+
+public class levelOne extends HeinousMap{
```
Comment (ministeve412) [inline core/src/net/heinousgames/game/dungeoncrawler/levelOne.java]: Classes have capital letters, method names are made like the one above... LevelOne
and a space after heinousmap before the bracket

**Comment by ministeve412** [comment]
🚢 merge ahoy 🚢

**Pull request merged**